conda-store-server/conda_store_server/alembic/versions/46bdf428642d_change_namespacerolemapping_and_.py

```diff
@@ -0,0 +1,59 @@
+"""Change NamespaceRoleMapping and Namespace
+
+Revision ID: 46bdf428642d
+Revises: b387747ca9b7
+Create Date: 2023-10-08 10:40:06.227854
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "46bdf428642d"
+down_revision = "b387747ca9b7"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
```

Comment:
This fails with SQLite because it doesn't have ALTER COLUMN.

I think I'll just drop/create the entire table since any data potentially stored there would be inconsistent anyway.